firefox-ios/nimbus-features/messaging/messaging-evergreen-messages.fml.yaml

```diff
@@ -53,3 +53,21 @@ import:
                 action: OPEN_URL
                 action-params:
                   url: https://www.macrumors.com
+
+              # Serves as an example of how microsurveys may look like
+              microsurvey-message:
+                surface: microsurvey
+                style: MICROSURVEY
+                trigger-if-all:
+                  - NEVER
+                title: "Help make printing in Firefox better. It only takes a sec."
```

Comment:
If this is to serve as an example, I'd rather this get replaced with the correct identifiers for the text for localization.

Reply:
Ah yes, the strings didn't go through the localization process, so I didn't see the localizable files yet, but I updated the configs and can verify / update later if any issues. The text does not have a fallback string.